packages/api/src/schema/db-schema.yaml

```
@@ -1018,6 +1019,39 @@ components:
           description:
             Timestamp (in milliseconds) at which the asset was deleted
           example: 1587667174725
+          index: true
+        status:
+          readOnly: true
+          type: object
+          additionalProperties: false
+          required:
+            - phase
+            - updatedAt
+          description: Status of the asset
+          properties:
+            phase:
+              type: string
+              description: Phase of the asset
+              index: true
+              enum:
+                - uploading
+                - waiting
+                - processing
+                - ready
+                - failed
+                - deleting
+                - deleted
+            updatedAt:
+              type: number
+              description:
+                Timestamp (in milliseconds) at which the asset was last updated
+              example: 1587667174725
+            progress:
+              type: number
+              description: Current progress of the task creating this asset.
+            errorMessage:
+              type: string
+              description: Error message if the asset creation failed.
```

We should not repeat the whole object, but just add the fields that are missing (`index`). The anti-pattern has spread in this file so it's hard to not follow it, but any time you see repetition it can be removed!
```suggestion
        status:
          properties:
            phase:
              index: true
```